public/template/messages.jade

```diff
@@ -10,9 +10,9 @@
       if _.isObject(activity)
         li
           small!= activity.content
-
-    if messages.items.length > 0
-      li.divider
-
+    if !messages.items.length
+      li
+        small= "You do not have any message yet"
```

Comment:
`messages` instead of `message`, and this should just be:

    small You do not have any messages yet.